src/sagemaker/image_uri_config/xgboost.json

```diff
@@ -125,6 +125,35 @@
                 "us-west-2": "246618743249"
             },
             "repository": "sagemaker-xgboost"
+        },
+        "1.2-1": {
```

Comment:
Did you also update the `version_aliases` on this file?

Reply:
@edwardjkim We should add `1.2` as an alias. 
@ChuyangDeng we can hold off on the other alias PR if there are conflicts with this PR.

Reply:
@iyerr3 Are we sure that we want to add `1.2` as an alias? Doesn't that kind of defeat the purpose of having the container version `-1` in `1.2-1`?

Reply:
@ChuyangDeng I talked to Rahul offline. We are going to hold off on `version_aliases` and revisit later.